Apigee/ManagementAPI/AbstractApp.php

```diff
@@ -833,7 +833,7 @@ protected static function loadFromResponse(AbstractApp &$obj, array $response, $
     protected static function loadCredentials(AbstractApp &$obj, array $credentials)
     {
         // Find the credential with the max issuedAt attribute which isn't expired.
-        if (count($credentials) > 0) {
+        if (!empty($credentials)) {
```

Comment:
This only checks to make sure the var is not empty, but isn't validating that it is an array.  Should this validate it is an array before trying to sort?

Reply:
Here we can be sure credentials is an array, since its required by the function declaration. This instance of count() doesnt have to be removed, but I generally went with using empty() instead of count ==0 or count > 0